hadoop-ozone/ozone-manager/src/main/java/org/apache/hadoop/ozone/om/request/key/acl/prefix/OMPrefixAclRequest.java

```diff
@@ -155,6 +160,14 @@ public OMClientResponse validateAndUpdateCache(OzoneManager ozoneManager, TermIn
     return omClientResponse;
   }
 
+  private void validatePrefixPath(String prefixPath) throws OMException {
+    if (!OzoneFSUtils.isValidName(prefixPath)) {
+      throw new OMException("Prefix path is not a valid path " +
+          "for prefix path: " + prefixPath,
```

Comment:
NIT: Maybe "Invalid prefix path name: " more concise?

Reply:
Thank you for the review. Updated. PTAL.